plugin.php

```diff
@@ -161,12 +161,17 @@ function get_destination_directory( $config = null ) {
 /**
  * Save the contents of a url to the static page destination.
  *
- * @param  string $contents
- * @param  string $url
- * @param  mixed  $config Option config object that will be passed to filters etc.
+ * @param $contents
+ * @param $url
+ * @param null $config
+ * @param int $page_id
+ * @param string $subsite_netstorage_path
```

Comment:
@dan-westall @tareiking Just pulled this up, docblock doesn't match function sig.